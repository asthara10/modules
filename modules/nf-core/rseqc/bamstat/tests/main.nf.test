nextflow_process {

    name "Test Process RSEQC_BAMSTAT"
    script "../main.nf"
    process "RSEQC_BAMSTAT"

    tag "modules"
    tag "modules_nfcore"
    tag "rseqc"
    tag "rseqc/bamstat"

    test("test_rseqc_bamstat") {


        when {
            process {
                """
                
                input[0] = [ [ id:'test' ], // meta map
              file(params.test_data['sarscov2']['illumina']['test_paired_end_sorted_bam'], checkIfExists: true)
            ]

                """
            }
        }

        then {
            assertAll(
                { assert process.success },
                { assert snapshot(process.out.versions).match("versions") }
                { assert snapshot(process.out).match() }
            )
        }

    }

}
