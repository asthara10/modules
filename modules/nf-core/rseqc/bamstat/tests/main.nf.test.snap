{
    "versions": {
        "content": [
            [
                "versions.yml:md5,300e8a9b6f8213d3621e53af09a9c728"
            ]
        ],
        "timestamp": "2023-11-24T12:43:07.306094"
    }
}